_posts/2022-02-01-engineering-product-design.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Engineering, Product and Design @ HolidayCheck"
+date: 2022-02-01 12:00:00 +0200
+categories: culture motivation product
+author_name: Álvaro Nistal Valbuena
+author_url : author/alvaronistalvalbuena/
+author_avatar: alvaronistalvalbuena
+read_time : 4
+excerpt: At HolidayCheck, we are proud of building and improving products and features that make holidays better. Our users trust HolidayCheck and expect from us the best content, great deals, top-of-the-notch user experience and a stable and fast platform.  
+feature_image: posts/2022-02-02-engineering-product-design/team.jpg
+
+---
+At HolidayCheck, we are proud of building and improving products and features that make holidays better. Our users trust HolidayCheck and expect from us the best content, great deals, top-of-the-notch user experience and a stable and fast platform.  
+
+Our product teams are the strongest unit of our product organisation. They are typically formed by 3-5 developers, 1 designer and 1 product manager and work independently to create a great experience for our users. Our developers, designers and product managers are those who know how to fulfil our goals and vision best and work together towards making better holidays. 
```

Comment:
`... and they work` - they missing

Reply:
`... fulfil our goals and vision best, and ` - comma missing

Reply:
Our developers, designers and product managers are those who know best how to fulfil our goals and vision and twork together towards making better holidays.